iam_users.tf

```diff
@@ -129,9 +129,7 @@ data "aws_iam_policy_document" "iam_self_admin_doc" {
     ]
 
     resources = [
-      # The MFA ARN is identical to that of the user, except that the
-      # text "user" is replaced by "mfa"
-      "${replace(aws_iam_user.user.*.arn[count.index], "user", "mfa")}",
+      replace(aws_iam_user.user[count.index].arn, "user", "mfa"),
```

Comment:
There's a deleted comment here FYI.

Reply:
Thanks!  I missed that.

Reply:
Comment added back in via commit 2e0a01dbdf3dac870bcbad7d068d8702dc7f42a2.